pinecone/sparse/splade/splade_quora.ipynb

```diff
@@ -0,0 +1,2438 @@
+{
```

Comment:
<blockquote>Unlike dense embeddings, which can be difficult to interpret, sparse embeddings map to tokens for easier interpretability.</blockquote><blockquote>See the <a href="https://github.com/pinecone-io/examples/blob/master/pinecone/sparse/splade/splade_vector_generation.ipynb" target="_blank">companion guide</a> to learn how to generate embeddings. &lt;--period</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>